### DISCUSSION TITLE
Slow forms/pages loading

### DISCUSSION THREAD

**Discussion by tmdoit** [Budibase/budibase#11270] [Help] [answered]
Hi,
Why each time when opening the form (e.g. Client Contact List app), assets are downloaded from host and not cached?
![image](https://github.com/Budibase/budibase/assets/85706044/69ce4a10-945f-4607-bb4e-2ba81dca185b)

**Comment by tmdoit** [comment]
The question is stupid? Did I miss something?

**Reply by ConorWebb96** [reply, marked as answer]
Hey @tmdoit,

Nope you are all good, I meant to respond to this yesterday evening but got a little sidetracked. When looking into this it seems to affect self-host and not cloud. It seems caching has been implemented into the cloud version of Budibase.

I've created this issue here as I believe this would be good if we do the same optimisation for self-host. You can follow the issue here; https://github.com/Budibase/budibase/issues/11307.

I hope this helps, thank you for bringing this to our attention.

**Reply by tmdoit** [reply]
Thank you! :)
